spec/orbital/request_spec/cvv_indicator_spec.rb

```diff
@@ -0,0 +1,164 @@
+require 'spec_helper'
```

Comment:
oh I didn't notice that. I've always used `rspec` to run all the tests. Removed the subdirectory in 05273d2